ui/webui/src/components/storage/DiskEncryption.jsx

```diff
@@ -303,14 +322,10 @@ export const DiskEncryption = ({
 }) => {
     const [password, setPassword] = useState(storageEncryption.password);
     const [confirmPassword, setConfirmPassword] = useState(storageEncryption.confirmPassword);
-    const [passwordStrength, setPasswordStrength] = useState("");
+    const [isValid, setIsValid] = useState(false);
```

Comment:
I don't see the purpose of maintaining the isValid as a state variable here. I suggest to directly use the setIsFormValid to invalidate or validate the form from the children component and remove this.

* DiskEncryption should always setIsFormValid(true) if the isEncrypted is false.
* The Password form fields should setIsFormValid(satisfied) where satisfied is the password rules result.

Reply:
Updated